src/index.html

```diff
@@ -5,8 +5,333 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <img
+        src="images/logo.png"
+        alt="Moyo logo"
+        class="logo"
+      >
+      <nav class="nav">
+
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link nav__link--color is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item" data-qa="nav-hover">
+            <a href="#laptops" class="nav__link">Laptops & Computers</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="grid">
+        <div
+          class="card"
+          data-qa="card"
+            >
+          <img
+            src="./images/imac.jpeg"
+            alt="imac"
+            class="card__photo"
+        >
+          <p class="card__description">
+          APPLE A1419 iMac 27" Retina <br>5K Monoblock (MNED2UA/A)
+          </p>
+          <p class="card__product-code">
+            Product code: 195434
+          </p>
+          <div class="card__rating">
+            <ul class="stars stars--4">
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+            </ul>
+            <p class="card__review">Reviews: 5</p>
+          </div>
+          <div class="card__price">
+            <p class="card__price card__price--text">Price:</p>
+            <p class="card__price card__price--value">$2,199</p>
+          </div>
+          <a
+            href="#"
+            class="card__link"
+            data-qa="card-hover"
+          >BUY
+          </a>
+        </div>
+        <div
+        class="card"
+          >
```

Comment:
```suggestion
        <div class="card">
```

Move each attribute to new line only if you have 3 or more attributes